.github/stale.yml

```diff
@@ -0,0 +1,59 @@
+# Configuration for probot-stale - https://github.com/probot/stale
```

Comment:
We are committed to review all PRs, so this file doesn’t seem to be appropriate.

Reply:
That's easy to remove

Reply:
I deleted it

Reply:
Thanks!